src/types/ToDoUser.ts

```diff
@@ -0,0 +1,12 @@
+export type ToDoUser = {
+  user: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  } | null;
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
```

Comment:
```suggestion
export type ToDoUser = {
  user: User | null;
  id: number;
  title: string;
  completed: boolean;
  userId: number;
};
```
just import the User type